suse_openstack_cloud

```diff
@@ -508,3 +508,9 @@ section_header "mkcloud"
 plog_files 0 /etc/motd
 plog_files 0 /root/mkcloud.config
 find_and_plog_files_0 /root -name \*.proposal
+###############################################################
+section_header "ardana"
+
+plog_files 0 /var/lib/ardana/.ansible/ansible.log
+find_and_plog_files_0 /var/lib/ardana/my_cloud/definition    -type f
+plog_files 0 /var/log/postgresql/*.log
```

Comment:
I'm pretty sure we collect already postgresql logs (also, Ardana uses galera/mariadb)

Reply:
I could not find the postgresql path that is included.
I included it as it was mentioned explicitly mentioned on the card

Reply:
I could not find the postgresql path that is included.
I included it as it was mentioned explicitly mentioned on the car